d2l-typography.html

```diff
@@ -18,6 +18,14 @@
 				@apply(--d2l-font-custom);
 			}
 
```

Comment:
If this is a new CSS class that you're introducing, you don't need the `vui-` version -- those are only there for other things because we renamed those classes from `vui-` to `d2l-` and wanted to keep supporting anyone using the old ones.